ldk/javascript/examples/self-test-loop/src/tests/user/index.ts

```diff
@@ -5,7 +5,40 @@ export const testJwt = (): Promise<boolean> =>
     user.jwt().then((token) => {
       if (token) {
         whisper.create({
-          label: 'User Aptitude JWT - No Email',
+          label: 'User Aptitude JWT - Default Claims',
```

Comment:
the diff here is a bit confusing. the change might make a little more sense if you view the whole file. This top test just had its name changed. The new test is below, and is basically the same test, except that `user.jwt` is called with the config parameter, like such as: `user.jwt({includeEmail: false})`